lumicks/pylake/force_calibration/tests/test_power_spectrum_calibration.py

```diff
@@ -195,6 +218,11 @@ def test_repr(reference_calibration_result):
         Bead diameter (um)   Bead diameter (um)                                        4.4
         Viscosity (Pa*s)     Liquid viscosity (Pa*s)                                   0.001002
         Temperature (C)      Liquid temperature (C)                                    20
+        Model                Calibration model (-)                                     PassiveCalibrationModel
+        Max iterations       Maximum number of function evaluations (-)                10000
+        Fit tolerance        Fitting tolerance (-)                                     1e-07
+        Points per block     Number of points per block (-)                            100
```

Comment:
For unitless parameters, it would be nice to set `unit=""` (empty string). It feels like a bit more natural way to represent unitless.

Reply:
Fair point 👍